config/_default/menus/menus.en.yaml

```diff
@@ -0,0 +1,1265 @@
+
+  - name: "Community Office Hours"
+    url: "developers/office_hours/"
+    parent: "dev_tools"
+    weight: 4
+
+  - name: "Write a Custom Check"
+    url: "developers/write_agent_check/"
+    parent: "dev_tools"
+    weight: 5
+
+  - name: "Write a Prometheus Check"
+    url: "developers/prometheus/"
+    parent: "dev_tools"
+    weight: 6
+
+  ## DEVELOPERS -  Integrations
+
+  - name: "Integrations"
+    url: "developers/integrations/"
+    parent: "dev_tools"
+    identifier: "dev_tools_integrations"
+    weight: 7
+  - name: "Create a new Integration"
+    url: "developers/integrations/new_check_howto/"
+    parent: "dev_tools_integrations"
+    weight: 701
+  - name: "Python"
+    url: "developers/integrations/python/"
+    parent: "dev_tools_integrations"
+    weight: 702
+  - name: "Legacy"
+    url: "developers/integrations/legacy/"
+    parent: "dev_tools_integrations"
+    weight: 703
+
+  - name: "Guides"
+    url: "developers/guide/"
+    parent: "dev_tools"
+    weight: 8
+
+#########
+## API ##
+#########
+
+  - name: "API"
+    url: "api/"
+    pre: "nav_api"
+    identifier: "api"
+    weight: 120000
+
+########################
+## ACCOUNT MANAGEMENT ##
+########################
+
+  - name: "Account Management"
+    url: "account_management/"
+    pre: "nav_account"
+    identifier: "account_management"
+    weight: 130000
+
+  - name: "Team management"
+    url: "account_management/team/"
+    parent: "account_management"
+    weight: 1
+
+  - name: "Organization settings"
+    url: "account_management/org_settings/"
+    parent: "account_management"
+    weight: 2
+
+  ## ACCOUNT MANAGEMENT - SAML
+
+  - name: "SSO with SAML"
+    url: "account_management/saml/"
+    identifier: "account_management_saml"
+    parent: "account_management"
+    weight: 3
+  - name: "Active Directory"
+    url: "account_management/saml/activedirectory/"
+    parent: "account_management_saml"
+    weight: 301
+  - name: "Auth0"
+    url: "account_management/saml/auth0/"
+    parent: "account_management_saml"
+    weight: 302
+  - name: "Azure"
+    url: "account_management/saml/azure/"
+    parent: "account_management_saml"
+    weight: 303
+  - name: "Google"
+    url: "account_management/saml/google/"
+    parent: "account_management_saml"
+    weight: 304
+  - name: "NoPassword"
+    url: "account_management/saml/nopassword/"
+    parent: "account_management_saml"
+    weight: 305
+  - name: "Okta"
+    url: "account_management/saml/okta/"
+    parent: "account_management_saml"
+    weight: 306
+  - name: "SafeNet"
+    url: "account_management/saml/safenet/"
+    parent: "account_management_saml"
+    weight: 307
+
+  - name: "Multi-org accounts"
+    url: "account_management/multi_organization/"
+    parent: "account_management"
+    weight: 4
+
+  - name: "Switching between orgs"
+    url: "account_management/org_switching/"
+    parent: "account_management"
+    weight: 5
+
+  - name: "Billing"
+    url: "account_management/billing/"
```

Comment:
Navigating to one of the links listed in the [Billing, Further Reading section](https://docs-staging.datadoghq.com/peter/async-load/account_management/billing/usage_details/) causes the nav to collapse. This doesn't happen on the [current site](https://docs.datadoghq.com/account_management/billing/#further-reading).

Reply:
fixed

Reply:
Nav still collapses but re-opens. Anyway for it to just stay open?

Reply:
not at this point because we've only put the async loading function on links in the nav, not on links in the main content area